packages/patterns/src/components/allow-overrides-modal.js

```diff
@@ -5,32 +5,36 @@ import {
 	__experimentalHStack as HStack,
 	__experimentalVStack as VStack,
 	Button,
+	Text,
```

Comment:
```suggestion
	__experimentalText as Text,
```

`Text` component is still experimental 😄